spec/std/socket/address_spec.cr

```diff
@@ -161,3 +161,91 @@ describe Socket::UNIXAddress do
     end
   end
 end
+
+describe Socket do
+  # Tests from libc-test:
+  # http://repo.or.cz/libc-test.git/blob/master:/src/functional/inet_pton.c
+  it ".ip?" do
+    # dotted-decimal notation
+    Socket.ip?("0.0.0.0").should be_true
```

Comment:
Maybe this should be on `Socket::Address`.

Reply:
Yep, that's actually a separate commit not included in this PR.